mypy.ini

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why `mypy.ini` and not just add these to `pyproject.toml`. Per [here](https://mypy.readthedocs.io/en/stable/config_file.html) this is a supported configuration setting for Mypy and it would ensure consistency with how we configure `isort` et al.

Reply:
Ok let me try it

Reply:
ok I did this and pushed a bit further to bring tox.ini into pyproject.toml as well